### ISSUE TITLE
[UWP][Accessibility] RichTextBlock TextRun with Action should use action title as Screen Reader text

### ISSUE THREAD

**Issue by nimercer** [microsoft/AdaptiveCards#3950]
# Platform

* UWP

# Author or host

Host (Cortana in Windows)

What version are you using? Ex: NuGet 1.0.2, or latest master, etc...

UWP Package 1.2.7

Explain your issue. If you just have a question, please post [on StackOverflow instead]

When rendering the below json: (displays as Hyperlink/Link Button)
```json
                {
                  "type": "TextRun",
                  "size": "small",
                  "text": "Open",
                  "selectAction": {
                    "type": "Action.OpenUrl",
                    "url": "{url}",
                    "title": "Open Alarms and Clock"
                  }
                }
```

The text for the a Screen Reader (Narrator) should be 'Open Alarms and Clock' as in the title of the OpenUrl action (so that Narrator text can be different to the display text).

Current result right now when Narrator reads the Hyperlink as 'Open' as its the button Text.

**Comment by shalinijoshi19** [comment]
@paulcam206 heads up; Let's prioritize the Cortana blockers higher up.. @nimercer where on your blocking priority list does this issue sit again? Thanks!

**Comment by nimercer** [comment]
@shalinijoshi19 / @paulcam206 

The accessibility team have classed this as a blocking issue so it's definitely high priority from our perspective. Given the limited time between now and General availability we won't be able to integrate the new package for our first release so we will look to get an exception for this release. If at all possible we'd like to integrate any fix into our next release which is expected to lock around the 22nd of May.

**Comment by shalinijoshi19** [comment]
@paulcam206 this was missing the 1.2.9 tag; Can we ensure this gets ported over to release/1.2 for inclusion in the UWP patch release?
